README.md

```diff
@@ -1662,6 +1662,8 @@ client.getMap('my-distributed-map').then(function (mp) {
 
 Hazelcast Map supports a Near Cache for remotely stored entries to increase the performance of read operations. See the [Near Cache section](#782-near-cache) for a detailed explanation of the Near Cache feature and its configuration.
 
+Hazelcast Map uses `MapListener` to listen to events which occur when entries are added to, updated in, merged in, evicted from or removed from the Map. See the [Map Listener section](#7521-map-listener) for information on how to create a map listener object and register it.
```

Comment:
Hazelcast Map uses `MapListener` to listen to the events that occur when the entries are added to, updated/merged  in or evicted/removed from the Map. ...